include/gz/sim/Link.hh

```diff
@@ -306,6 +306,20 @@ namespace gz
                                  const math::Vector3d &_force,
                                  const math::Vector3d &_torque) const;
 
+      /// \brief Add a wrench expressed in world coordinates and applied to
+      /// the link at the given offset from its center of mass. This wrench
+      /// is applied for one simulation step.
+      /// \param[in] _ecm Mutable Entity-component manager.
+      /// \param[in] _force Force to be applied expressed in world coordinates
+      /// \param[in] _offset The point of application of the force expressed
```

Comment:
I'm not trying to be nitpicky, but the frame of expression for these quantities has always been a point of confusion, so I want to be a little more careful. I see that the offset is relative to the COM, but it's expressed in the link frame. Wouldn't it be more consistent if it was expressed in the COM frame? i.e, if you have the following SDF
```xml
<model name="test">
  <link name="base">
    <pose>1 0 0   1.57 0 0</pose>
    <inertial>
      <pose>0 1 0    0 1.57 0</pose>
      ...
      </inertial>
   </link>
</model>

With the current function, we'd be ignoring the pose of the `<inertial>` tag.

Reply:
I agree, expressing the offset in the COM frame is more intuitive and consistent. Changed it in 336b119.